open-api/swagger.yaml

```diff
@@ -588,6 +588,20 @@ paths:
                 $ref: '#/components/schemas/Version'
         '404':
           description: Version not available.
+  /health:
```

Comment:
As it's spring boot functionality that we document here I recommend mention that in the description and add a link to the official documentation.

Reply:
good idea!